.github/workflows/update_pbuilder_cache.yml

```diff
@@ -59,23 +59,53 @@ jobs:
           echo "::endgroup"
 
       - name: run pdebuild
+        id: pdebuild
+        continue-on-error: true
         run: |
-          pdebuild --buildresult ~/daily_build/ubuntu --architecture ${{ matrix.arch }} -- --distribution ${{ matrix.distroname }}
+          mkdir -p ~/daily_build/ubuntu
+          pdebuild --logfile ~/daily_build/pdebuild_log.txt --buildresult ~/daily_build/ubuntu --architecture ${{ matrix.arch }} -- --distribution ${{ matrix.distroname }}
+
+      - name: generate badge
+        run: |
+          pip install pybadges setuptools
+          if [ '${{ steps.pdebuid.outcome }}' == 'success' ]; then
+            python3 -m pybadges --left-text=test --right-text=success --right-color=green > ~/daily_build/pdebuild_result.svg
+          else
+            python3 -m pybadges --left-text=test --right-text=failure --right-color=red > ~/daily_build/pdebuild_result.svg
+          fi
```

Comment:
Good PR :+1: ,  Maybe we should change ```steps.pdebuid.outcome``` to ```steps.pdebuild.outcome```. Right?

Reply:
Updated.